kaldb/src/test/java/com/slack/kaldb/writer/kafka/KaldbKafkaConsumerTest.java

```diff
@@ -223,6 +223,7 @@ public void testConsumeMessagesBetweenOffsets() throws Exception {
       // Assign doesn't create a consumer group.
       assertThat(kafkaServer.getConnectedConsumerGroups()).isEqualTo(0);
     }
+
```

Comment:
Looks like an errant return here

Reply:
this was the only file that changes because of the new google format version bump